system_meta/management/commands/update_product_image_data.py

```diff
@@ -0,0 +1,49 @@
+from django.core.management.base import BaseCommand
+
+from system_meta.models import Product
+
+
+class Command(BaseCommand):
+    """
+    A management command to update image_metadata for all Product objects
+    Example usage: python manage.py update_product_image_data --product_id 1
+    """
+
+    help = "Update image_metadata for all Product objects"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--product-id",
+            type=int,
+            help="The ID of the product to update",
+        )
+        parser.add_argument(
```

Comment:
If we're taking an `sku` here it should also take the system - SKUs aren't unique except within an integrated system.